Samples/Samples/ViewModel/FilePickerViewModel.cs

```diff
@@ -136,12 +136,16 @@ async Task<FileResult> PickAndShow(PickOptions options)
 
                 if (result != null)
                 {
-                    Text = $"File Name: {result.FileName}";
+                    var size = await GetStreamSizeAsync(result);
+
+                    Text = $"File Name: {result.FileName} ({size:0.00} KB)";
 
                     if (result.FileName.EndsWith("jpg", StringComparison.OrdinalIgnoreCase) ||
+                        result.FileName.EndsWith("jpeg", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
Not new to your PR but since you're fixing it :)
A file named `This-is-not-a-jpeg` would hit that condition.
You should use https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getextension?view=net-5.0